server/cli_common/utils.py

```diff
@@ -215,14 +213,58 @@ def create_dask_client(args):
       empty string to start one locally
 
     """
+    import dask
     scheduler_address = args.scheduler_address
 
+    if scheduler_address in ('thread', 'threaded', 'threads'):
```

Comment:
@manthey I suggest using just one value instead of three. Maybe `multithreading` and `multiprocessing`